templates/tag.twig

```diff
@@ -8,26 +8,25 @@
 				background-size: cover;
 			}
 			.tag_template {
-				padding-top: 100px;
+				padding-top: 200px;
 			}
-			.tag_template .tag_header_container {
+			.tag_template > .container {
 				color: white;
 			}
-			.category_link {
+			.page-header > a {
```

Comment:
@ashish173 
This is the css that I had used for the code-generated Campaign page. I guess it should be in presentation layer as well.